### ISSUE TITLE
[Translation] 8_8_cleaning_up.md

### ISSUE THREAD

**Issue by Johnny4Fun** [cfsamson/book-exploring-async-basics#46]
`Lastly we clean up after ourselves by joining all our threads ...`

so what we need to clean up is just waiting the threads finished? Actually it is `after ourselves`  that confuses me. Maybe I have asked a similar quetion, but I am still confused.

**Comment by cfsamson** [comment]
>Lastly we clean up after ourselves by joining all our threads ...

Yeah, well I think you're right "after ourselves" doesn't really add any meaningful here, it's more of a way of just saying "Lastly, we clean up when we're finished by joining all our threads".

The reason for this is that we'll catch any panics that might happen in a child thread and we make sure all values are are dropped (which releases any resources they might hold on to).
